torch/csrc/jit/passes/onnx.cpp

```diff
@@ -140,8 +140,14 @@ void ToONNX(std::shared_ptr<tracer::TracingState>& state) {
       node_list outputs;
       if (py::isinstance<Node>(raw_output)) {
         outputs = node_list{py::cast<Node*>(raw_output)};
+        outputs[0]->setDebugName(n->debugName());
+        outputs[0]->setSourceLocation(n->getSourceLocation());
       } else {
         outputs = py::cast<std::vector<Node*>>(raw_output);
+        for (Node* node : outputs) {
+          node->setDebugName(n->debugName());
+          node->setSourceLocation(n->getSourceLocation());
```

Comment:
Can you also add this here: https://github.com/jamesr66a/pytorch/blob/28c0ceee1e62945016a2cd73e07bd6b7c28f9539/torch/csrc/jit/ir.h#L534

Reply:
Yeah, I'm wondering if we shouldn't have just called `copyAttributes` here. Also note that we copy types over here: https://github.com/jamesr66a/pytorch/blob/369cd1980eb7ac74ca349c21b97040856d4cf3ea/torch/csrc/jit/passes/onnx.cpp#L80 which may be a better place for this logic, as there are multiple calls to `setOutputs`.